### ISSUE TITLE
`typing.TypeAlias` & `typing.Annotated`

### ISSUE THREAD

**Issue by AlexWaygood** [PyCQA/flake8-pyi#149]
Right now, this plugin will not allow the following code:

```python
from typing import Annotated

SpecialInt = Annotated[int, "Very important metadata"]  # Y026 Use typing_extensions.TypeAlias for type aliases
```

Is this correct behaviour? What _is_ the interaction between PEP 593 & PEP 613?

**Comment by AlexWaygood** [comment]
I suppose this is intended behavior, since `SpecialInt = int` should also be annotated with `TypeAlias` according to Y026.

**Comment by JelleZijlstra** [comment]
Yes, that's correct. The code you posted should use TypeAlias.
